Yosemite/Yosemite/Model/Orders/OrderDateRangeFilter.swift

```diff
@@ -41,3 +41,14 @@ public enum OrderDateRangeFilterEnum: Codable, Hashable, CaseIterable {
         case custom
     }
 }
+
+public struct ProductFilter: Codable, Hashable {
+    public let id: Int64
+    public let name: String
+
+    public init(id: Int64,
+         name: String) {
+        self.id = id
+        self.name = name
+    }
+}
```

Comment:
Is this redundant?

Reply:
Yup, I missed this obvious thing https://github.com/woocommerce/woocommerce-ios/pull/12100/commits/7643455ab28449fdd1c3d99067de76c98eabfea6